parquet/benches/arrow_reader.rs

```diff
@@ -1307,6 +1386,17 @@ fn add_benches(c: &mut Criterion) {
     );
     group.finish();
 
+    // primitive int64 / logical uint64 benchmarks
+    let mut group = c.benchmark_group("arrow_array_reader/UInt64Array");
+    bench_primitive::<Int64Type>(
```

Comment:
I think this is meant to be 
```suggestion
    bench_primitive::<UInt64Type>(
```